symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/src/main/java/com/symphony/bdk/core/api/invoker/ApiClient.java

```diff
@@ -16,18 +15,18 @@
   /**
    * Invoke API by sending HTTP request with the given options.
    *
-   * @param <T> Type
-   * @param path The sub-path of the HTTP URL
-   * @param method The request method, one of "GET", "POST", "PUT", "HEAD" and "DELETE"
-   * @param queryParams The query parameters
-   * @param body The request body object
+   * @param <T>          Type
```

Comment:
Looks like different code styles are used, we should use the same everywhere

Reply:
We should use : https://github.com/SymphonyOSF/appcentral/blob/master/.editorconfig